webui/react/src/components/HyperparameterSearchModal.tsx

```diff
@@ -626,15 +639,19 @@ const HyperparameterSearchModal = ({ closeModal, experiment, trial }: Props): JS
         <div className={css.spacer} />
         <Row>
           <Button onClick={handleCancel}>Cancel</Button>
-          <Button disabled={validationError} type="primary" onClick={handleOk}>
+          <Button
+            disabled={validationError}
```

Comment:
```suggestion
            disabled={validationError || isSubmitting}
```

Reply:
I'm hesitate to do that because I don't believe other modals disable the button for submitting, or do we recently change that? I'd prefer modals to look similar.